Qdk feb rel notes (#720)
MicrosoftDocs/quantum-docs-pr: articles/relnotes/index.md

For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).

## Version 0.xx.xxxx.xxxx

*Release date: February 27th, 2020*

This release contains the following:

> - New Quantum Machine Learning library, for more information go to our [QML docs page](https://docs.microsoft.com/quantum/libraries/machine-learning/?view=qsharp-preview)

Reviewer: The 10×–20× speedup is indeed an IQ# issue, and affects loading of packages; that is, it's not a general speedup. Since the katas have not been updated to the newest version of IQ#, there may not be a change in performance there for this release.

Reviewer 2: Define "for this release" :-) It will be once we update the katas, for which the release needs to be out.

> - Bug fixes to the Quantum Katas, resulting in a 10-20x performance increase

Reviewer 2: It is my understanding that the fixes are not in the katas but in the QDK itself (was it iqsharp?) - the perf improvement just manifests on the katas?

```suggestion
-  IQ# bug fixes, resulting in a 10-20x performance increase on the Quantum Katas
```

Author: thanks Mariia!

> -  IQ# bug fixes, resulting in a 10-20x performance increase on the Quantum Katas

Reviewer: The 10×–20× speedup is indeed an IQ# issue, and affects loading of packages; that is, it's not a general speedup. Since the katas have not been updated to the newest version of IQ#, there may not be a change in performance there for this release. The other IQ# bug fix was provided by @crazy4pi314, and fixes little-endian vs big-endian in the enhanced Jupyter-based state display.

Reviewer 2: Per my deleted comment, I'll update the katas once the release is public.

Author: @cgranade, is there a suggestion to update the phrasing here, or is your comment just clarifying where the speedup is coming from?

Reviewer: My concern is that the speedup is not in general; performance isn't 10× faster for all IQ# operations, just the package loading. I'd suggest clarifying in the release notes so that users aren't disappointed when not _everything_ is faster.

Author: @bettinaheim update this to say "up to" so marking as resolved & merging the PR

> -  IQ# bug fixes, resulting in a 10-20x performance increase on the Quantum Katas

Reviewer 3: ```suggestion
-  IQ# bug fixes, resulting in up to a 10-20x performance increase on the Quantum Katas
```

See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+is%3Aclosed), [compiler](https://github.com/microsoft/qsharp-compiler/pulls?q=is%3Apr+is%3Aclosed), [runtime](https://github.com/microsoft/qsharp-runtime/pulls?q=is%3Apr+is%3Aclosed), [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+is%3Aclosed) and [Katas](https://github.com/microsoft/QuantumKatas/pulls?q=is%3Apr+is%3Aclosed).  

## Version 0.10.2001.2831

*Release date: January 29th, 2020*

This release contains the following:

> - New Microsoft.Quantum.SDK NuGet package which will replace Microsoft.Quantum.Development.Kit NuGet package upon creating new projects. Microsoft.Quantum.Development.Kit NuGet package will continue to be supported for existing projects. 

Author: ```suggestion
- New Microsoft.Quantum.SDK NuGet package which will replace Microsoft.Quantum.Development.Kit NuGet package when creating new projects. Microsoft.Quantum.Development.Kit NuGet package will continue to be supported for existing projects. 
```

> - Support for Q# compiler extensions, enabled by the new Microsoft.Quantum.SDK NuGet packge, for more information see the [documentation on Github](https://github.com/microsoft/qsharp-compiler/tree/master/src/QuantumSdk#extending-the-q), the [compiler extensions sample](https://github.com/microsoft/qsharp-compiler/tree/master/examples/CompilerExtensions) and the [Q# Dev Blog]()

Reviewer 3: ```suggestion
- Support for Q# compiler extensions, enabled by the new Microsoft.Quantum.SDK NuGet packge, for more information see the [documentation on Github](https://github.com/microsoft/qsharp-compiler/tree/master/src/QuantumSdk#extending-the-q-compiler), the [compiler extensions sample](https://github.com/microsoft/qsharp-compiler/tree/master/examples/CompilerExtensions) and the [Q# Dev Blog]()
```

> - Support for Q# compiler extensions, enabled by the new Microsoft.Quantum.SDK NuGet packge, for more information see the [documentation on Github](https://github.com/microsoft/qsharp-compiler/tree/master/src/QuantumSdk#extending-the-q-compiler), the [compiler extensions sample](https://github.com/microsoft/qsharp-compiler/tree/master/examples/CompilerExtensions) and the [Q# Dev Blog]()

Reviewer 3: ```suggestion
- Support for Q# compiler extensions, enabled by the new Microsoft.Quantum.SDK NuGet packge, for more information see the [documentation on Github](https://github.com/microsoft/qsharp-compiler/tree/master/src/QuantumSdk#extending-the-q-compiler), the [compiler extensions sample](https://github.com/microsoft/qsharp-compiler/tree/master/examples/CompilerExtensions) and the [Q# Dev Blog](https://devblogs.microsoft.com/qsharp/extending-the-q-compiler/)
```
